README.md

```diff
@@ -220,3 +222,11 @@ Example:
   }
 }
 ```
+
+#### SSL Certificate issue (behind proxy)
+
+Due to proxy redirection you may face SSL certificate issue on Yarn when installing the Otter project locally.
```

Comment:
you may add a quote of the type of error you can get to help people find this with search tools
> Request Error: self-signed certificate in certificate chain